sdk/schemaregistry/schema-registry/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0-beta.1 (2023-01-11)
```

Comment:
for my own info: do we want to add a disclaimer about the json schema versions at all?

Reply:
Good point, added in https://github.com/Azure/azure-sdk-for-js/pull/24444/commits/2739abb12719ca29e26cc5f0fa5b5522326160ee.